archstrike/ps1encode/PKGBUILD

```diff
@@ -0,0 +1,40 @@
+# Maintainer: ArchStrike <team@archstrike.org>
+# Contributor: strayArch <j.a.stronz@gmail.com>
+
+buildarch=1
+
+_pkgname=ps1encode
+pkgname=${_pkgname}-git
```

Comment:
if pkgname ps1encode-git, it needs to be in folder `archstrike/ps1encode-git`
i'd also prefer if you put it into `archstrike-testing` instead